### ISSUE TITLE
Format line numbers in an editor friendly way when checking types

### ISSUE THREAD

**Issue by Systho** [s-panferov/awesome-typescript-loader#128]
Would it be possible to format the type check warnings in an editor friendly way ?

I think the line responsible for doing this is that one : https://github.com/s-panferov/awesome-typescript-loader/blob/73f575d0c448a0f19a5da542220fe905c07e4674/src/checker-runtime.ts#L239

I've tried the following change and now I can jump directly to the correct line in my editors (jetbrains, sublime and atom )

```
console.error(`[${ instanceName }] ${colors.red(diagnostic.file.fileName)}:${line + 1}:${character + 1}  :\n    ${colors.red(message)}`);
```

**Comment by s-panferov** [comment]
@Systho 	thanks, I'll change this

**Comment by s-panferov** [comment]
Should be fixed in v0.17.0.
